Engine.UnitTests/EmbeddedPropertyTests.cs

```diff
@@ -146,12 +146,11 @@ public void OnPostRun(TestStepPostRunEventArgs step)
             }
         }
         
-        public class RepeatMixin : ITestStepPostRunMixin, ITestStepPreRunMixin
+        public class RepeatMixin2 : ITestStepPostRunMixin, ITestStepPreRunMixin
```

Comment:
Why was this renamed?

Reply:
Just to avoid clashing with a mixin defined by Basic Mixins. Its just a unit test, so it should not cause a problem.